scripts/lib.star

```diff
@@ -692,6 +717,7 @@ def copy_packages_for_docker_step():
             'package',
         ],
         'commands': [
+            'ls dist/*.tar.gz*',
```

Comment:
Is this supposed to stay or was it for debugging?

Reply:
@AgnesToulet It's for debugging, but I kept it because I thought it could still be useful? I figure it doesn't hurt when you're checking a build, to see which files get copied.